docs/UserGuide.md

```diff
@@ -87,15 +92,15 @@ Examples:
 
 Adds a person to FriendDex.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [b/BIRTHDAY] [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/DATETIME [t/TAG]…​`
```

Comment:
I think you have to write the `b/` prefix stands for. I think we can leave birthday there. Users should know to supply a `date` from the examples below.